crates/turbopack-css/src/chunk/writer.rs

```diff
@@ -18,10 +21,16 @@ pub async fn expand_imports(
         "".to_string(),
     )];
     let mut external_imports = vec![];
+    let mut imported_chunk_items: HashSet<CssChunkItemVc> = HashSet::default();
+    let mut composed_chunk_items: HashSet<CssChunkItemVc> = HashSet::default();
```

Comment:
They should share a single set

Reply:
I thought about it, but imported chunk items can also have opening/closing text that composed items don't (which I'm not sure the purpose of), so what happens if we add a composed chunk item, and *then* the same, imported chunk item, with these open/close attributes?

Reply:
I've updated the code and the tests to account for the possibility of two imports of the same file with different import attributes.